### PR TITLE
created .prettierrc file with some standard guidelines

### PR DESCRIPTION
**Pull request by Muh-Hasan**

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"useTabs": true,
```
Comment (mukarramali) [inline .prettierrc.json:2]: Please add tab size as 2

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	"semi": true,
+	"singleQuote": false,
```
Comment (mukarramali) [inline .prettierrc.json:4]: singleQuote being true or false is always debatable, but let's stick to [airbnb](https://github.com/airbnb/javascript#strings--quotes) style guides which uses single quote

File: .prettierrc.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	"bracketSpacing": true,
+	"jsxBracketSameLine": true,
+	"arrowParens": "always",
+	"filePath": "./",
+	"requirePragma": false,
```
Comment (mukarramali) [inline .prettierrc.json:9]: this is by default false, so we can avoid writing it here

**Review by mukarramali — CHANGES_REQUESTED** [review]

File: .prettierrc.json (hunk 1 above)
Comment (mukarramali) [reply .prettierrc.json:2]: And useTabs as false

**Comment by mukarramali** [comment]
Please link issues you are working on

**Comment by Muh-Hasan** [comment]
> Please link issues you are working on

ok

File: index.js [hunk 4]
```diff
@@ -1,98 +1,96 @@
...
+	predictInteraction,
+	logAction,
 } = require("./vacabot/helpers");
 const {
-  actionCheckVacationBalance,
-  actionDenyVacationRequest,
-  actionOpenCreateVacation,
-  actionSubmitVacationRequest,
-  actionUpdateManager
+	actionCheckVacationBalance,
```
Comment (mukarramali) [inline index.js:9]: They weren't updated from 4 tab size to 2 tab size

**Review by mukarramali — CHANGES_REQUESTED** [review]

File: index.js (hunk 4 above)
Comment (Muh-Hasan) [reply index.js:9]: let me check

File: package-lock.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (mukarramali) [inline package-lock.json:1]: There is already a yarn.lock file, please use that. Yarn is preferred over npm for multiple reasons, you can also read, quite interesting.

**Review by mukarramali — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 5 above)
Comment (Muh-Hasan) [reply package-lock.json:1]: yes, sure

File: .prettierrc.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mukarramali) [inline .prettierrc.json:1]: please use `.prettierrc`as file name

**Review by mukarramali — CHANGES_REQUESTED** [review]

File: .prettierrc.json (hunk 6 above)
Comment (Muh-Hasan) [reply .prettierrc.json:1]: ok

Comment (mukarramali) [reply .prettierrc.json:1]: right now it is: `.prettierrc.json`

**Pull request merged**